clients/src/main/java/org/apache/kafka/clients/consumer/NoOffsetForPartitionException.java

```diff
@@ -61,4 +73,11 @@ public TopicPartition partition() {
         return partitions;
     }
 
+    /**
+     * returns a map of all partitions for which no offset are defined, and their respective fetch states
+     * @return all partitions with no offset and their respective fetch states
+     */
+    public Map<TopicPartition, SubscriptionState.FetchState> getPartitionFetchStates() {
```

Comment:
following the convention, this method should be called partitionFetchStates

Reply:
Changing the name